treasurenetprotocol/treasurenet
# Modify the Makefile file, change the coverage of unit testing, and remove useless modules to make the code look more concise

This pull request includes several changes to improve the unit test coverage and code cleanliness in the project:

    Makefile Updates:
        Modified the Makefile to enhance the unit test coverage reporting. This change ensures that all relevant code paths are being tested and accurately reported.
    Test Case Enhancements in /x/gravity:
        Reviewed and updated all test files under the /x/gravity directory. These updates include adding new test cases to increase coverage, modifying existing tests to improve their effectiveness, and removing redundant or obsolete tests.
    Test Case Updates in /x/evm:
        Similar to /x/gravity, the test files under /x/evm have been revised. This involves writing new tests, adjusting existing ones, and eliminating unnecessary tests to maintain a high-quality test suite.
    Code Clean-up:
        Removed unused modules and files from the /x/ directory. This clean-up effort helps to reduce clutter, improve maintainability, and ensure that the codebase remains lean and focused.

How to Test:

To test these changes, follow these steps:

    Clone the repository and checkout this branch.
    Run make test to execute the updated unit tests and verify the coverage report.
    Optionally, you can run specific tests for /x/gravity and /x/evm directories individually to validate the changes in more detail.
